lib/taskana-core/src/main/java/pro/taskana/task/api/TaskService.java

```diff
@@ -490,4 +490,28 @@ TaskComment getTaskComment(String taskCommentId)
    */
   BulkOperationResults<String, TaskanaException> setPlannedPropertyOfTasks(
       Instant planned, List<String> taskIds);
+
+  /**
+   * Cancels a task.
+   *
+   * @param taskId the id of the task to cancel.
+   * @return the updated task.
+   * @throws TaskNotFoundException if the Task with Id TaskId is not found
+   * @throws InvalidStateException if the task is not in state READY or CLAIMED
+   * @throws NotAuthorizedException if the current user is not authorized to see the task
+   */
+  Task cancelTask(String taskId)
+      throws TaskNotFoundException, InvalidStateException, NotAuthorizedException;
+
+  /**
+   * Terminates a task.
```

Comment:
Terminates a task.
Termination is a administrative action to complete a task. This is typically done by an administration to correct any technical issue.

Reply:
I added the description